examples/adminapi.py

```diff
@@ -260,6 +260,35 @@ def example_delete_acls(a, args):
             raise
 
 
+def example_incremental_alter_configs(a, args):
+    """ Incrementally alter configs, keeping non-specified
+    configuration properties with their previous values.
+
+    Input Format : ResourceType1 ResourceName1 Key=Operation:Value;Key2=Operation2:Value2
+    ResourceType2 ResourceName2 ...
+
+    Example: TOPIC T1 compression.type=SET:lz4;cleanup.policy=ADD:compact TOPIC T2 compression.type=SET:gzip ...
+    """
+    resources = []
+    for restype, resname, configs in zip(args[0::3], args[1::3], args[2::3]):
+        resource = ConfigResource(restype, resname)
```

Comment:
Let's create this using constructor to help with the example on how to use the constructor as well?

Reply:
In this case given that it's not a static list but it's created dynamically I think using the constructor make the example more difficult to read. I add a comment about using the constructor